netbox/provider.go

```diff
@@ -63,6 +63,7 @@ func Provider() *schema.Provider {
 			"netbox_cluster_type":         resourceNetboxClusterType(),
 			"netbox_cluster":              resourceNetboxCluster(),
 			"netbox_contact":              resourceNetboxContact(),
+			"netbox_contact_group":        resourceNetboxContact(),
```

Comment:
I don't have permissions to push to their fork / branch and therefore directly contribute to this PR but what about this?

```suggestion
			"netbox_contact_group":        resourceNetboxContactGroup(),
```

Reply:
Oh right. My bad :D

As for the change you mentioned: Yes, that seems to be wrong, but the tests fail for another reason.

I will look into this MR later (tm).

Reply:
I don't want to tell how long I debugged until I found in wireshark that the HTTP calls are made to /contacts instead of /contact-groups.
Then I was like "damn, I remember this". D'oh! :see_no_evil: :D

Merge is now underway. No clue how the tests suite passes locally.

Fun fact: It does not pass locally if you only run the single test via `-run TestAccNetboxContactGroupDataSource_basic` to the `go test` make target.